### DISCUSSION TITLE
Website is down

### DISCUSSION THREAD

**Discussion by adesugbaa** [livewire/livewire#6405] [4. General]
Not sure where to report this but the [site](https://laravel-livewire.com) is showing the beautiful 500 | Server Error page.

**Comment by Dev-RaheelAhmad** [comment]
I have also noticed occasional server errors on the old domain. (https://laravel-livewire.com)
Alternatively, you can make use of this new domain: 👉🏻 https://livewire.laravel.com

**Reply by inmanturbo** [reply]
The new domain only hosts docs for 3x and volt (which depends on 3x)
